File: project_euler/prob001.py
#If we list all the natural numbers below 10 that are multiples of 3 or 5, we get 3, 5, 6 and 9.
#The sum of these multiples is 23.
#Find the sum of all the multiples of 3 or 5 below 1000.

menor = 0
maior = 1000
passo = 1
soma = 0

while menor < maior:
    if menor % 3 == 0 or menor % 5 == 0:
        soma += menor
    menor += passo

print soma

File: project_euler/prob001.c
#include <stdio.h>

int main (void)
{
  int menor , maior , passo , soma;
menor = 0;
maior = 1000;
passo = 1;
soma = 0;

while (menor < maior)
{
if (menor % 3 == 0 || menor % 5 == 0) {
soma += menor;
}
menor += passo;
}
printf("A soma dos multiplo de 3 e 5 menor que 1000 e %d\n", soma);
 return 0;
}
